README.md

```diff
@@ -209,30 +209,12 @@ ansible-playbook create-vm-instances-for-geth.yml                   \
     --user      $user
 ```
 
-### Authorizing a user account on the build server to access the clusters
-This step must be performed separately for every user of the build server who needs to be able to access other parts of the project infrastructure on Google Cloud with `kubectl` or `gcloud`.
-It can be performed by user himself or an admin who can impersonate him with `sudo`.
+All the instructions below assume that you're using local playbooks to run build and deployment commands on a new server.
 
 The `$cluster` variable decides about deployment to specific environment but also which server will be use for deployment process.
 If you want it for `mainnet`, `testnet` or `staging` environments you must create and configure the `concent-deployment` server.
 This behavior applies to all playbooks, except building container process (`build-test-and-push-containers.yml`).
```

Comment:
Why is this now in the section about deploying the geth nodes?

I think your intention here was to remove this section completely but after a rebase some new lines jumped in. Please fix it.

Reply:
I think that this part should stay. It informs about the assumptions of the instructions.

Reply:
But it's in the wrong place. It's meant for someone interested in the sections below but you put it in a section about the geth client.

I would suggest this:
- Rename the `Deployment` section to `Initializing the deployment environments`
- I would start a new `Deployment` section just above `Building containers and cluster configuration`.
- Put the text that applies to all the subsections (starting from `All the instructions below assume ...`) at the beginning of that new section.

BTW, this text implies that there's a separate deployment server for each environment. Is this how it works now? I hope not.

Reply:
It is an outdated change.
> BTW, this text implies that there's a separate deployment server for each environment. Is this how it works now? I hope not.

All suggestions are added in the properly PR: #382 . Please check it.